From 4071a44f8ce6cfc7eabe5f68db9958ed9a714c1d Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 11 Aug 2016 16:44:42 -0700
Subject: [PATCH] joyent/node-verror#29 No documentation on how to Install this
 package Reviewed by: Todd Whiteman <todd.whiteman@joyent.com> Approved by:
 Todd Whiteman <todd.whiteman@joyent.com>

---
 CHANGES.md | 1 +
 README.md  | 4 ++++
 2 files changed, 5 insertions(+)

diff --git a/CHANGES.md b/CHANGES.md
index f6dc25c..4839f1a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,4 +2,5 @@
 
 * #10 better support for extra properties on Errors
 * #11 make it easy to find causes of a particular kind
+* #29 No documentation on how to Install this package
 * #36 elide development-only files from npm package
diff --git a/README.md b/README.md
index 902bdf3..5d34a53 100644
--- a/README.md
+++ b/README.md
@@ -30,6 +30,10 @@ The classes here are:
 
 # Quick start
 
+First, install the package:
+
+    npm install verror
+
 If nothing else, you can use VError as a drop-in replacement for the built-in
 JavaScript Error class, with the addition of printf-style messages:
 
-- 
2.21.0

